### PR TITLE
DEVDOCS-4811:[update] Update the batch GET and PUT limits for PriceLists

### PR DESCRIPTION
**Pull request by bc-tgomez**
Update the batch limit rates for GET and PUSH calls.

- /v3/pricelists/{pricelist_id}/records
- /v3/pricelists/{pricelist_id}/records/{variant_id}
- /v3/pricelists/{pricelist_id}/records/{variant_id}/{currency_code}

### CODE REVIEW

**Comment by bc-tgomez** [comment]
Hey there @megdesko, would you check this PR and make sure I captured your changes?

File: reference/price_lists.v3.yml [hunk 1]
```diff
@@ -1387,7 +1402,7 @@ paths:
 
         * Batch updates are supported by this endpoint, meaning that several price objects can be updated in one request. This allows you to do the same work as many individual requests to singleton endpoints
         * Batch requests support up to 1,000 items per request.
-        * Up to 2 concurrent batch upsert requests are supported with this API. Running more than the allowed concurrent requests in parallel on the **same store** will cause a 429 error and your additional requests will fail. You are encouraged to run requests sequentially with as many records per request as possible, in order to maximize performance.
```
Comment (megdesko) [inline reference/price_lists.v3.yml:1390]: This one stays 2.  It's a batch endpoint for updating up to 1K items in one go and is a heavy lift.  Apologies if I wasn't clear about that earlier.

File: reference/price_lists.v3.yml [hunk 2]
```diff
@@ -1654,7 +1683,7 @@ paths:
         Returns *Price List Records* using the variant ID. Will also contain currency records.
         
         **Notes**
-        * Supports up to 50 simultaneous GET requests. Running more than the allowed number of requests concurrently on the same store will result in a `429` status error and your additional requests will fail.
+        * Supports up to 40 simultaneous GET requests. Running more than the allowed number of requests concurrently on the same store will result in a `429` status error and your additional requests will fail.
```
Comment (bc-traciporter) [inline reference/price_lists.v3.yml:1686]:
```suggestion
        * Supports up to 40 simultaneous GET requests. Running more than the allowed number of requests concurrently on the same store will result in a `429` status error, and your additional requests will fail.
```

File: reference/price_lists.v3.yml [hunk 3]
```diff
@@ -1859,15 +1888,33 @@ paths:
...
   '/pricelists/{price_list_id}/records/{variant_id}/{currency_code}':
     parameters:
       - $ref: '#/components/parameters/Accept'
     get:
       tags:
         - Price Lists Records
       summary: Get a Price Record by Currency Code
-      description: Returns a *Price List Record* using the currency code. Optional parameters can be used.
+      description: |- 
+        Returns a *Price List Record* using the currency code. Optional parameters can be used.
```
Comment (bc-traciporter) [inline reference/price_lists.v3.yml:1914]:
```suggestion
        Returns a *Price List Record* using the currency code. You can use optional parameters.
```

File: reference/price_lists.v3.yml [hunk 4]
```diff
@@ -1859,15 +1888,33 @@ paths:
...
+
+        **Notes**
+        * Supports up to 40 simultaneous GET requests. Running more than the allowed number of requests concurrently on the same store will result in a `429` status error and your additional requests will fail.
```
Comment (bc-traciporter) [inline reference/price_lists.v3.yml:1917]:
```suggestion
        * Supports up to 40 simultaneous GET requests. Running more than the allowed number of requests concurrently on the same store will result in a `429` status error, and your additional requests will fail.
```

File: reference/price_lists.v3.yml [hunk 5]
```diff
@@ -2052,11 +2099,30 @@ paths:
...
     put:
       tags:
         - Price Lists Records
       summary: Set Price List Record by Currency Code
-      description: Creates or updates a*Price List Record* using the currency code.
+      description: |-
+        Creates or updates a *Price List Record* using the currency code.
+
+        **Notes**
+        * Supports up to 40 simultaneous PUT requests. Running more than the allowed number of requests concurrently on the same store will result in a `429` status error and your additional requests will fail.
```
Comment (bc-traciporter) [inline reference/price_lists.v3.yml:2125]:
```suggestion
        * Supports up to 40 simultaneous PUT requests. Running more than the allowed number of requests concurrently on the same store will result in a `429` status error, and your additional requests will fail.
```

**Review by bc-traciporter — COMMENTED** [review]
I added a comma and removed the passive voice in a sentence.

**Pull request merged**